### ISSUE TITLE
Update VA health connect phone number label

### ISSUE THREAD

**Issue by mmiddaugh** [department-of-veterans-affairs/va.gov-cms#17750]
## Description
To add in Veteran understanding, IVC program leadership has asked for the VA health connect label to be updated to incorporate the Clinical Contact Center

This should be done during the same sprint as [FE: Add VA health chat and VA health connect to the Manage your health online box #17436](https://app.zenhub.com/workspaces/sitewide-facilities-639f5253e4b702a32376339e/issues/gh/department-of-veterans-affairs/va.gov-cms/17436), if possible 

## Tasks
Update VA health connect phone label from` VA health connect` to `VA Health Connect Clinical Contact Center`

- in Facility Locator search results
- in VAMC Location and contact info blocks on VAMC facility pages
- in VAMC Location and contact info blocks on VAMC system pages
- in VAMC Location and contact info blocks on VAMC location pages

## Screenshots

<details>
<summary> Facility Locator search results </summary>

<img width="244" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBK0UyQkE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--3c271e09c25a552aeb7225dc353d580f0b1a3d00/image.png" alt="image.png" />
</details>

<details>
<summary>VAMC Location and contact info block </summary>

<img width="587" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBK0kyQkE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--4dfb02a312525e8dc4ee5b2c935776ba63a75cb4/image.png" alt="image.png" />

</details>

**Comment by jilladams** [comment]
@Agile6MSkinner I noticed in the board that this ticket has Sprint 8 assigned, but we didn't pull it in during planning. Could you verify whether this does / doesn't need to get into current sprint, along with the Health Chat/Connect link ticket? (#17436 )

**Comment by Agile6MSkinner** [comment]
I had wanted to pull it into the sprint based on the note about getting it done at the same time as #17463. I think someone said we were waiting on something for it to be ready. I took the sprint label off.

**Comment by jilladams** [comment]
@mmiddaugh @Agile6MSkinner Should we plan this ticket for Sprint 9, or is this a no-op now that the parent epic is closed?

**Comment by jilladams** [comment]
Per Michelle, we shouldn't close this ticket / it may come back up, but is not a priority right now.
